src/OpenTelemetry.Instrumentation.StackExchangeRedis/Implementation/RedisProfilerEntryToActivityConverter.cs

```diff
@@ -62,6 +66,13 @@ public static Activity ProfilerCommandToActivity(Activity parentActivity, IProfi
                 activity.SetTag(SemanticConventions.AttributeDbSystem, "redis");
                 activity.SetTag(StackExchangeRedisCallsInstrumentation.RedisFlagsKeyName, command.Flags.ToString());
 
+                var message = GetMessageValue(command);
+                if (!string.IsNullOrEmpty(message))
+                {
+                    // Example: "db.operation": [0]:GET key1
+                    activity.SetTag(SemanticConventions.AttributeDbOperation, message);
```

Comment:
From semantic conventions doc, db.operation is required only if db.statement is not applicable. (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#call-level-attributes)
Since we already populate the db.statement, we need to have good reason for populating db.operation considering its not free to obtain the value for it.

Reply:
Thanks a lot for review.
I've tested and had have next results
###### Read data from REDIS
```
db.redis.database_index=0
db.redis.flags=1024
db.statement=GET
peer.service=xx.xx.xx.xx:6379
net.peer.ip=xx.xx.xx.xx
db.operation=[0]:GET sso:templates:String:USERRESET (RedisValueProcessor)
```
###### Delete key in REDIS
```
db.redis.database_index=0
db.redis.flags=1028
db.statement=UNLINK
peer.service=xx.xx.xx.xx:6379
net.peer.ip=xx.xx.xx.xx
db.operation=[0]:UNLINK sso:templates:String:USERRESET (DemandZeroOrOneProcessor)
```
###### Write JSON to REDIS
```
db.redis.database_index=0
db.redis.flags=1028
db.statement=SETEX
peer.service=xx.xx.xx.xx:6379
net.peer.ip=xx.xx.xx.xx
db.operation=[0]:SETEX sso:templates:String:USERRESET (BooleanProcessor)
```
So, `db.operation` contains REDIS operation, db number and key. Semantic conventions doc contains next text
> When setting this to an SQL keyword, it is not recommended to attempt any client-side parsing of `db.statement` just to get this property, but it should be set if the operation name is provided by the library being instrumented.

> For **Redis**, the value provided for db.statement SHOULD correspond to the syntax of the Redis CLI. If, for example, the `HMSET command` is invoked, `"HMSET myhash field1 'Hello' field2 'World'"` would be a suitable value for db.statement.

`db.statement` does not contain key and value. `db.operation` contains key but does not contain value. It is much more information to analyze!
How would you like an idea if I add parsing `db.operation` to get Redis CLI-like to match the semantic conventions doc?